system-int-test/test/cmr/system_int_test/access_control/permissions_route_test.clj

```diff
@@ -0,0 +1,40 @@
+(ns cmr.system-int-test.access-control.permissions-route-test
+  (:require
+   [cheshire.core :as json]
+   [clj-http.client :as client]
+   [clojure.string :as string]
+   [clojure.test :refer :all]
+   [cmr.access-control.test.util :as u]
+   [cmr.system-int-test.system :as system]
+   [cmr.system-int-test.utils.ingest-util :as ingest]
+   [cmr.system-int-test.utils.url-helper :as url]))
+
+(deftest permission-get-and-post-request-test
+  (let [save-basic-collection (fn [short-name]
+                                  (u/save-collection {:entry-title (str short-name " entry title")
+                                                      :short-name short-name
+                                                      :native-id short-name
+                                                      :provider-id "PROV1"}))
+        coll1 (save-basic-collection "coll1")
+        coll2 (save-basic-collection "coll2")
+        coll3 (save-basic-collection "coll3")
+        coll4 (save-basic-collection "coll4")
+        permissions-url (url/access-control-permissions-url)
+        post-data-body (str "{ \"user_type\" : \"guest\", \"concept_id\" : ["
+                         "\"" coll1 "\", "
+                         "\"" coll2 "\", "
+                         "\"" coll3 "\", "
+                         "\"" coll4 "\""
+                         "] }")]
```

Comment:
Would prefer if this was marshaled into JSON with cheshire.

Reply:
Fixed